galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -2352,6 +2418,30 @@ The excerpt below is an example of how this Galera parameter can be triggered fr
    SET GLOBAL wsrep_provider_options = 'socket.ssl_reload=1';
 
 
+.. _`sync_binlog`:
+.. rst-class:: section-heading
+.. rubric:: ``sync_binlog``
+
+.. index::
+   pair: wsrep Provider Options; sync_binlog
+
+Synchronously flush binary log to disk after every #th event. Use 0 (default) to disable synchronous flushing.
+
+.. csv-table::
+   :class: doc-options
+
+   "Default Value", "0"
+   "Dynamic", "Yes"
+   "Initial Version", ""
+
+The excerpt below is an example of how this Galera parameter might look in the configuration file, using the maximum value of the parameter:
+
+.. code-block:: ini
+
+   SET GLOBAL wsrep_provider_options = 'sync_binlog=4294967295';

```

Comment:
no, sync_binlog is not a wsrep_provider_option 

https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_sync_binlog

Reply:
Updated.